### DISCUSSION TITLE
Does JOOQ support jsonbSet() for Postgres ?

### DISCUSSION THREAD

**Discussion by apptio-speravali** [jOOQ/jOOQ#17412] [Q&A] [answered]
was looking at the code and jsonbSet is not yet supported. 

```java
@NotNull
    @Support({ MARIADB_10_2, MYSQL_5_7, ORACLE21C, SQLITE, SQLSERVER2016 })
    public static Field<JSONB> jsonbSet(Field<JSONB> field, @Stringly.Param String path, Object value) {
        return new JSONBSet(field, Tools.field(path), Tools.field(value));
    }
```

Currently we are using DSL.field and building SQL, and works fine. But wanted to see if there is any timeline to support DSL.jsonbSet for Postgres

**Comment by lukaseder** [comment, marked as answer]
Thanks for your message. I don't think it's possible to emulate MySQL's more sophisticated version of this function in PostgreSQL, see the discussions here:

- https://github.com/jOOQ/jOOQ/issues/13951 (look for "PostgreSQL")
- https://stackoverflow.com/q/73581878/521799

PostgreSQL's version doesn't support JSON paths to identify the location where the manipulation should take place. Let me know if you know of a way to emulate the MySQL version of the function
